src/main/java/org/c4sg/controller/ProjectController.java

```diff
@@ -222,19 +222,19 @@ public String retrieveImage(@ApiParam(value = "Project id to get image for", req
     @ApiOperation(value = "Deletes a project's images")
     public String deleteImage(@ApiParam(value = "Project id to delete image for", required = true)
```

Comment:
This method is an UPDATE operation not a DELETE operation, because you're not deleting the resource just updating it. So change the this method.